File: backend/backend/ENTITY/Entities/UserExercise.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace LifeGuideProject.API.ENTITY.Entities
{
    [Table("UserExercise", Schema = "public")]
    public class UserExercise
    {
        [Key]
        public int id { get; set; }
        public string userEmail { get; set; }
        public int exerciseId { get; set; }
        public string action { get; set; }

    }
}

File: frontend/src/app/Models/userExercise.ts
export class UserExercise{
    public id:number;
    public userEmail : string;
    public exerciseId : number;
    public action : string;

    constructor(id:number,userEmail:string,exerciseId:number,action : string){
        this.id = id;
        this.userEmail = userEmail;
        this.exerciseId = exerciseId;
        this.action = action;
    }
}